cmd/orb-server/startcmd/params.go

```diff
@@ -85,10 +85,15 @@ const (
 	didAliasesFlagShorthand = "a"
 	didAliasesFlagUsage     = "Aliases for this did method. " + commonEnvVarUsageText + didAliasesEnvKey
 
-	casURLFlagName      = "cas-url"
-	casURLFlagShorthand = "c"
-	casURLEnvKey        = "CAS_URL"
-	casURLFlagUsage     = "The URL of the Content Addressable Storage(CAS). " + commonEnvVarUsageText + casURLEnvKey
+	casTypeFlagName      = "cas-type"
+	casTypeFlagShorthand = "c"
+	casTypeEnvKey        = "CAS_TYPE"
+	casTypeFlagUsage     = "The type of the Content Addressable Storage(CAS): ipfs or webcas" + commonEnvVarUsageText + casTypeEnvKey
```

Comment:
Followup - an orb server can support both IPFS and WebCas.

Reply:
We already support both at the same time for reading. One domain supports one type of cas (ipfs or local) for writing.

Reply:
Maybe the name should be changed to "local" instead of "webcas" here, since I think WebCAS is the mechanism for retrieving data in another server's CAS

Reply:
Actually it is local - I forgot to change help message when I changed it from "webcas" to "local". I will fix info message.

Reply:
Done

Reply:
Actually my point was that a domain can support both for writing as well.

Reply:
Created new issue: https://github.com/trustbloc/orb/issues/399